lib/ecto/migration/boot_migrator.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Ecto.Migration.BootMigrator do
+  @moduledoc """
+  A process you can add to your Application Supervisor to run migrations. This will attempt to run migrations, and then silently shut its self down.
+
+  Add the following to the top of your application children spec:
+     {Ecto.Migration.BootMigrator, otp_app: :my_app}
+
+  Optionally pass `:skip` to skip them. 
+
+  #TODO More config
+  """
+  use GenServer
+
+  # Callbacks
+  @impl true
+  def init(opts) do
+    app = Keyword.fetch!(opts, :otp_app)
+    repos = Application.fetch_env!(app, :ecto_repos)
```

Comment:
allow repos as option, fallback to ecto_repos? Alternatively, caller always passes it?

Reply:
also trading two options for `{Ecto.Migrator, repos: Application.fetch_env!(:my_app, :ecto_repos)}`